compiler/luci-interpreter/src/core/Tensor.cpp

```diff
@@ -45,7 +45,14 @@ void Tensor::writeData(const void *data_ptr, size_t data_size)
 {
   const size_t element_size = getDataTypeSize(element_type());
   const int32_t num_elements = shape().num_elements();
-  if (data_size != num_elements * element_size)
+  if (element_type() == loco::DataType::S4)
+  {
+    if (data_size != (num_elements + 1) / 2)
+    {
+      throw std::invalid_argument("Invalid data size.");
+    }
+  }
+  else if (data_size != num_elements * element_size)
```

Comment:
`getDataTypeSize(element_type())` will return as same as result of `int8_t`
So, I guess we do not need this conversion(or packing).

cc. 
https://github.com/Samsung/ONE/blob/a031542b9e67f6fca3bf89b8a78666040fc4bf08/compiler/loco/include/loco/IR/DataTypeTraits.h#L38-L61